From f7656005900f3c07450303afa66151b6a1bb9599 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 17 Feb 2012 14:40:01 +0100
Subject: [PATCH 409/434] qxl: drop vram bar minimum size

There is no reason to require a minimum size of 16 MB for the vram.
Lower the limit to 4096 (one page).  Make it disapper completely would
break guests.
---
 hw/qxl.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hw/qxl.c b/hw/qxl.c
index b910337..d71c94d 100644
--- a/hw/qxl.c
+++ b/hw/qxl.c
@@ -1595,8 +1595,8 @@ static int qxl_init_common(PCIQXLDevice *qxl)
     init_qxl_rom(qxl);
     init_qxl_ram(qxl);
 
-    if (qxl->vram_size < 16 * 1024 * 1024) {
-        qxl->vram_size = 16 * 1024 * 1024;
+    if (qxl->vram_size < 4096) {
+        qxl->vram_size = 4096;
     }
     if (qxl->revision == 1) {
         qxl->vram_size = 4096;
-- 
1.7.10

